seal/__init__.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import
+
+from .models import SealableManager, SealableModel
```

Comment:
You can't expose models of a Django app at root level since 1.7. There's a ticket to allow abstract models to by pass this rule but it didn't get much traction from what I remember.